plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -18,6 +18,7 @@
 import time
 import traceback
 from collections import OrderedDict
+from re import S
```

Comment:
Is this an intentional new import? I couldn’t find where this was now being used.

Reply:
oops, this looks like a typo - let me correct it! Thank you for pointing this out.

Reply:
Fixed it.